packages/utils/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "@canonical/typescript-config-base",
   "compilerOptions": {
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "types": ["node"]
```

Comment:
I believe we shouldn't need the `node` type here now, wince you changed the timer type in `throttle.ts` to `ReturnType<>`.

Reply:
```suggestion
    "types": []
```